src/e2e-test/java/io/cdap/plugin/cloudsqlpostgresql/stepsdesign/CloudSqlPostGreSql.java

```diff
@@ -158,20 +180,20 @@ public void validateMandatoryPropertyErrorFor(String property) {
 
     @Then("Enter Reference Name & Database Name with Test Data")
```

Comment:
with Test Data -> with valid test data.

Also step description does not match method name

Reply:
done